deis-dev/manifests/deis-minio-rc.yaml

```diff
@@ -18,8 +18,24 @@ spec:
         - name: deis-minio
           image: quay.io/deisci/minio:v2-beta
           imagePullPolicy: Always
+          env:
+            - name: HEALTH_SERVER_PORT
+              value: "8082"
           ports:
             - containerPort: 9000
+            - containerPort: 8082
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8092
+            initialDelaySeconds: 30
+            timeoutSeconds: 1
```

Comment:
@mboersma seems like https://github.com/deis/workflow/issues/344 is specifically targeted at supporting RDS, but in this case the health readiness and liveness probe endpoints _always_ make an s3 request to the local minio server. IMO if that takes longer than 1 second, the service should be considered down. Thoughts?